src/main/java/Main.java

```diff
@@ -1,15 +1,106 @@
+import display_use_case.DisplayInputBoundary;
+import display_use_case.DisplayInteractor;
+import display_use_case.DisplayPresenter;
+import Screens.*;
+import analysis_use_case.Analysis;
+import delete_inventory_use_case.DeleteInventoryInputBoundary;
+import delete_inventory_use_case.DeleteInventoryInteractor;
+import delete_inventory_use_case.DeleteInventoryPresenter;
+import entities.Inventory;
+import export_use_case.ExportInputBoundary;
+import export_use_case.ExportPresenter;
+import export_use_case.ExporterInventory;
+import generate_order_use_case.OrderingInteractor;
+import generate_order_use_case.OrderingPresenter;
+import import_use_case.ImportInventory;
+import import_use_case.ImportPresenter;
+import new_item_use_case.NewItemInputBoundary;
+import new_item_use_case.NewItemInteractor;
+import new_item_use_case.NewItemPresenter;
+import receive_order_use_case.ReceivingInteractor;
+import receive_order_use_case.ReceivingPresenter;
+import remove_item_use_case.RemoveItemInputBoundary;
+import remove_item_use_case.RemoveItemInteractor;
+import remove_item_use_case.RemoveItemPresenter;
+import update_item_quantity_use_case.UpdateItemQtyInputBoundary;
+import update_item_quantity_use_case.UpdateItemQtyInteractor;
+import update_item_quantity_use_case.UpdateItemQtyPresenter;
 
-import Screens.MainCreationUI;
-import entities.*;
-import useCases.UpdateItemQuantity;
-
-import java.sql.SQLOutput;
-import java.util.ArrayList;
-import java.util.Collections;
 import java.util.HashMap;
 
 public class Main {
     public static void main(String[] args) {
-        new MainCreationUI();
+
+        //create a new blank inventory
+        Inventory inv = new Inventory("New Inventory");
+
+        //map of controllers
+        //ex. controllers.get("newItemController") = newItemController
+        HashMap<String, Object> controllers = new HashMap<>();
+
+        DisplayPresenter displayPresenter = new DisplayInventoryUpdater();
```

Comment:
Added setup for use cases in main. Creates controller, presenter, and interactor, and stores them in a HashMap.